apps/ideaspace/src/components/common/IdeaCard/IdeaCard.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import Link from 'next/link';
+import { atoms } from '@devlaunchers/components/src/components';
+import {
+  ImgButton,
+  StatuBox,
+} from './StyledIdeaCard';
+import commentSvg from '../../../images/comment.svg';
+import timeSvg from '../../../images/time.svg';
+
+function IdeaCard({ cards, cardType }) {
+  const [tag, setTag] = useState('');
+  const [tagColor, setTagColor] = useState('');
+  const [updated, setUpdated] = useState('');
+  React.useEffect(() => {
+    if (cards.statu == "submitted"){
+      setTag("submitted");
+      setTagColor("#A7D5E3");
```

Comment:
@alicefate can we get these colors from our `theme`?

Reply:
@dbradham I don't know how to use theme. `setTagColor(${({ theme }) => theme.colors.NEUTRAL_1});` and `setTagColor(theme.colors.NEUTRAL_1});` will jump error.